bootstrapper/internal/nodelock/nodelock_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package nodelock
+
+import (
+	"io"
+	"sync"
+	"sync/atomic"
+	"testing"
+
+	"github.com/edgelesssys/constellation/v2/internal/attestation/vtpm"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTryLockOnce(t *testing.T) {
+	t.Run("should lock exactly once", func(t *testing.T) {
+		assert := assert.New(t)
```

Comment:
```suggestion
func TestTryLockOnce(t *testing.T) {
	assert := assert.New(t)
```
nit: Don't need a table driven test structure for non table driven tests